packages/x-components/src/x-modules/related-tags/components/__tests__/related-tag.spec.ts

```diff
@@ -1,79 +1,64 @@
-import { RelatedTag } from '@empathyco/x-types';
-import { DeepPartial } from '@empathyco/x-utils';
-import { mount, VueWrapper } from '@vue/test-utils';
+import { mount } from '@vue/test-utils';
 import { nextTick } from 'vue';
-import { Store } from 'vuex';
-import { getXComponentXModuleName, isXComponent } from '../../../../components/x-component.utils';
-import { XComponentAPI } from '../../../../plugins/x-plugin.types';
-import { RootXStoreState } from '../../../../store/store.types';
-import { getRelatedTagsStub } from '../../../../__stubs__/related-tags-stubs.factory';
+import { getXComponentXModuleName, isXComponent } from '../../../../components';
+import { getRelatedTagsStub } from '../../../../__stubs__';
 import { getDataTestSelector, installNewXPlugin } from '../../../../__tests__/utils';
-import { WireMetadata } from '../../../../wiring/wiring.types';
+import { WireMetadata } from '../../../../wiring';
 import { relatedTagsXModule } from '../../x-module';
-import RelatedTagComponent from '../related-tag.vue';
-import { XPlugin } from '../../../../plugins/x-plugin';
+import RelatedTag from '../related-tag.vue';
+import { XPlugin } from '../../../../plugins';
 
 describe('testing related tag item component', () => {
   function renderRelatedTag({
     relatedTag = getRelatedTagsStub()[0],
     template = '<RelatedTag :relatedTag="relatedTag" />'
-  }: RenderRelatedTagOptions = {}): RenderRelatedTagsAPI {
-    const store = new Store<DeepPartial<RootXStoreState>>({});
-    installNewXPlugin({ store });
-    // Manually re-installing the xModule
-    XPlugin.registerXModule(relatedTagsXModule);
-
-    const wrapperTemplate = mount(
+  } = {}) {
+    const wrapper = mount(
       {
-        props: ['relatedTag', 'highlightCurated'],
-        components: {
-          RelatedTag: RelatedTagComponent
-        },
+        components: { RelatedTag },
         template
       },
       {
-        store,
-        global: { plugins: [installNewXPlugin()] },
-        props: { relatedTag }
+        global: { plugins: [installNewXPlugin({ initialXModules: [relatedTagsXModule] })] },
+        props: { relatedTag, highlightCurated: false }
       }
     );
-    const wrapper = wrapperTemplate.findComponent(RelatedTagComponent);
+    const wrapperRelatedTag = wrapper.findComponent(RelatedTag);
 
     return {
       wrapper,
-      wrapperTemplate,
+      wrapperRelatedTag,
```

Comment:
```suggestion
      wrapperRelatedTag: wrapper.findComponent(RelatedTag);,
```
And delete the const definition above

Reply:
Done